dts-shared-services/openshift/templates/backup-mariadb/backup-mariadb-build.yaml

```diff
@@ -0,0 +1,74 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: ${NAME}-build-template
+  creationTimestamp: null
+objects:
+  - kind: ImageStream
+    apiVersion: v1
+    metadata:
+      name: ${NAME}
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: ${NAME}
+      labels:
+        app: ${NAME}
+    spec:
+      triggers:
+        - type: ImageChange
+        - type: ConfigChange
+      runPolicy: Serial
+      source:
+        type: Git
+        git:
+          uri: ${GIT_REPO_URL}
+          ref: ${GIT_REF}
+        contextDir: ${SOURCE_CONTEXT_DIR}
+      strategy:
+        type: Docker
+        dockerStrategy:
+          from:
+            kind: DockerImage
+            name: ${BASE_IMAGE_FOR_BUILD}
+          dockerfilePath: ${DOCKER_FILE_PATH}
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:${OUTPUT_IMAGE_TAG}
+parameters:
+  - name: NAME
+    displayName: Name
+    description: The name assigned to all of the resources.  Use 'backup-{database name}' depending on your database provider
+    required: true
+    value: backup-postgres
+  - name: GIT_REPO_URL
+    displayName: Git Repo URL
+    description: The URL to your GIT repo.
+    required: true
+    value: https://github.com/BCDevOps/backup-container.git
+  - name: GIT_REF
+    displayName: Git Reference
+    description: The git reference or branch.
+    required: true
+    value: master
+  - name: SOURCE_CONTEXT_DIR
+    displayName: Source Context Directory
+    description: The source context directory.
+    required: false
+    value: /docker
+  - name: DOCKER_FILE_PATH
+    displayName: Docker File
+    description: The path and file of the docker file defining the build.  Choose either 'Dockerfile' for Postgres builds or 'Dockerfile_Mongo' for MongoDB builds or 'Dockerfile_MSSQL' for MSSQL builds.
+    required: false
+    value: Dockerfile
+  - name: OUTPUT_IMAGE_TAG
+    displayName: Output Image Tag
+    description: The tag given to the built image.
+    required: true
+    value: latest
+  - name: BASE_IMAGE_FOR_BUILD
+    displayName: FROM Image Tag
+    description: Base image to build from.  Docker creds or Artificatory setup may be needed to alleviate docker rate-limiting
+    required: true
+    value: docker.io/centos/postgresql-12-centos7:20200917-804ef01
```

Comment:
don't we have a bcgov docker registry `docker-remote.artifacts.developer.gov.bc.ca/` that we can use for this?

Reply:
No, this actually points to a Fedora repo (I just fixed it) and Artifactory is not configured to cache it.